utils/db_helper.js

```diff
@@ -59,7 +74,7 @@ const GET_BET_INVESTORS =
   'SELECT buyer, direction, SUM(investmentamount) AS amount FROM amm_interactions WHERE bet = $1 GROUP BY buyer, direction;';
 
 const UPDATE_BALANCE_OF_USER =
-  'INSERT INTO token_balances (owner, symbol, last_update, balance) VALUES($1, $2, $3, $4) ON CONFLICT (owner, symbol) DO UPDATE SET last_update = $3, balance = token_balances.balance + $4 RETURNING balance;';
+  'UPDATE account SET balance = balance + $3 WHERE owner_account = $1 AND symbol = $2 AND account_namespace = $4 RETURNING balance;';
```

Comment:
It will fail in `trading-engine` if you try to mint or burn an account that does not exist. Are you suggesting we should change that logic and create accounts on mint and burn in case they do not exist?